src/pages/components/components.json

```
@@ -1,4 +1,18 @@
 [
+  {
+    "title": "svelte-client-router",
+    "url": "https://arthurgermano.github.io/svelte-client-router/#/svelte-client-router",
+    "npm": "https://www.npmjs.com/package/svelte-client-router",
```

This should be only the NPM package name, not the NPM URL
```suggestion
    "npm": "svelte-client-router",
```